src/lrf.rs

```diff
@@ -1082,22 +1082,24 @@ fn wiener_stripe_filter<T: Pixel>(
   let limit = (1 << (bit_depth + 1 + WIENER_BITS - round_h)) - 1;
 
   let mut work: [i32; MAX_SB_SIZE + 7] = [0; MAX_SB_SIZE + 7];
+  let vfilter3 = 128
```

Comment:
This looks weird. I guess it would be better to exclude this piece of code from being formatted by rustfmt at all. It would be also nice to report this issue to Clippy authors.